src/sentry/integrations/msteams/utils.py

```diff
@@ -54,3 +55,376 @@ def get_channel_id(organization, integration_id, name):
+        }
+    else:
+        resolve_action = {
+            "type": "Action.ToggleVisibility",
+            "title": "Resolve",
+            "targetElements": resolve_targets,
+        }
+
+    if status == GroupStatus.IGNORED:
+        ignore_action = {
+            "type": "Action.Submit",
+            "title": "Stop Ignoring",
+            "data": {"actionType": "unignore"},
+        }
+    else:
+        ignore_action = {
+            "type": "Action.ToggleVisibility",
+            "title": "Ignore",
+            "targetElements": ignore_targets,
+        }
+
+    assign_text = "Assign"
+    assign_action = {
+        "type": "Action.ToggleVisibility",
+        "title": assign_text,
+        "targetElements": assign_targets,
+    }
+
+    return {
+        "type": "ColumnSet",
+        "columns": [
+            {
+                "type": "Column",
+                "items": [{"type": "ActionSet", "actions": [resolve_action]}],
+                "width": "stretch",
+            },
+            {
+                "type": "Column",
+                "items": [{"type": "ActionSet", "actions": [ignore_action]}],
+                "width": "stretch",
+            },
+            {
+                "type": "Column",
+                "items": [{"type": "ActionSet", "actions": [assign_action]}],
+                "width": "stretch",
+            },
+        ],
+    }
+
+
+def build_incident_resolve_card():
+    title_card = {
+        "type": "TextBlock",
+        "size": "Large",
+        "text": "Resolve",
+        "weight": "Bolder",
+        "id": "resolveTitle",
+        "isVisible": False,
+    }
+
+    input_card = {
+        "type": "Input.ChoiceSet",
+        "value": "0",
+        "id": "resolveInput",
+        "isVisible": False,
+        "choices": [
+            {"title": "Immediately", "value": "0"},
+            {"title": "In the current release", "value": "1"},
+            {"title": "In the next release", "value": "2"},
+        ],
+    }
+
+    submit_card = {
+        "type": "ActionSet",
+        "id": "resolveSubmit",
+        "isVisible": False,
+        "actions": [
+            {"type": "Action.Submit", "title": "Resolve", "data": {"actionType": "resolve"}}
+        ],
+    }
+
+    return [title_card, input_card, submit_card]
+
+
+def build_incident_ignore_card():
+    title_card = {
+        "type": "TextBlock",
+        "size": "Large",
+        "text": "Ignore until this happens again...",
+        "weight": "Bolder",
+        "id": "ignoreTitle",
+        "isVisible": False,
+    }
+
+    input_card = {
+        "type": "Input.ChoiceSet",
+        "value": "0",
+        "id": "ignoreInput",
+        "isVisible": False,
+        "choices": [
+            {"title": "1 time", "value": "0"},
+            {"title": "10 times", "value": "1"},
+            {"title": "100 times", "value": "2"},
+            {"title": "1,000 times", "value": "3"},
+            {"title": "10,000 times", "value": "4"},
```

Comment:
@KKaempen The value here should just be the numerical value of the number of times